experanto/data.py

```diff
@@ -3,20 +3,36 @@
 from collections import namedtuple
 from collections.abc import Iterable
 from pathlib import Path
+
 import numpy as np
 import torch
 from torch.utils.data import Dataset
 
 from .experiment import Experiment
-from .interpolators import ImageTrial
+from .interpolators import ImageTrial, VideoTrial
 
 
 class Mouse2pChunkedDataset(Dataset):
-    def __init__(self, root_folder: str, sampling_rate: float, chunk_size: int) -> None:
+    def __init__(
+        self,
+        root_folder: str,
+        sampling_rate: float,
+        chunk_size: int,
+        rescale: bool = False,
+        keep_nans: bool = False,
```

Comment:
the other kwargs can be dealt with the config_dict that I describe below see below